### ISSUE TITLE
Consider removing publishing from alevin-fry step

### ISSUE THREAD

**Issue by jashapiro** [AlexsLemonade/scpca-nf#154]
Since we do not use the output from `alevin-fry` as a checkpoint, we might not want to include it in the `interal` outputs as we proceed toward wider use.

To make this change, we would remove the following line from the module: 
 https://github.com/AlexsLemonade/scpca-nf/blob/0112aac248f0015d3b87602d7ceb387bc593f15b/modules/af-rna.nf#L48

Similarly, we should probably remove `publishDir` directives from the feature mapping: https://github.com/AlexsLemonade/scpca-nf/blob/main/modules/af-features.nf We don't currently checkpoint feature mapping at all (though we could)

If we need to look at any of these files for development or debugging, the `publishDir` directive can always be added back temporarily.

One further thought I had was also to rename `internal` to `checkpoints` or similar to make it more clear what the function of that folder is.

**Comment by allyhawkins** [comment]
Yes, we discussed this on Friday and are in agreement so will slate it for this sprint.

**Comment by allyhawkins** [comment]
In working on this again and also a brief chat with Deepa during WAYWO, I was thinking that maybe we want to make writing out these files optional instead of removing them completely. There could be scenarios in which people might prefer to have the output from alevin-fry directly which is what this offers rather than having it as an RDS file. It also has a few other logs that if someone was troubleshooting a bad library *may* be helpful, or at least I could imagine a user wanting to have those files to determine if they are helpful. Although technically one could dig around in the work directory... it is nice to have things organized in an understandable way if one was debugging or looking for the alevin-fry outputs. 

In looking at the nextflow docs, I believe we could make it optional by creating a parameter, `save_fry_outs` (or a better name...) that we store as `false` and then use that to tell the process to either publish the files or not with the [`enabled` option for `publishDir`](https://www.nextflow.io/docs/latest/process.html#publishdir). 
```
publishDir "${params.outdir}/checkpoints/af/${meta.library_id}", enabled: params.save_fry_outs
```
Then by using `--save_fry_outs` at the command line a user could tell the workflow to save the alevin fry output. I wanted to post this here for any additional thoughts before working on any of the other changes (which includes updating the user description of the output files) related to this issue. 

Also I did a quick test of this and it works as expected.

**Comment by allyhawkins** [comment]
Closed by #169.
